routes/course_routes.js

```diff
@@ -0,0 +1,43 @@
+var express = require('express');
+var handleError = require(__dirname + '/../lib/handle_server_error.js');
+var jsonParser = require('body-parser').json;
```

Comment:
call the function here, so you don't have to call it each time below. Remember the json function constructs and returns the middleware we're after, there's no need to do it each time we want to use it.